### PR TITLE
Update aes.h

### PR DESCRIPTION
**Pull request by RosZad**
Minor documentation improvements:
Standardized brief file description.
Split returns.
Minor fixes.

### CODE REVIEW

File: include/mbedtls/aes.h [hunk 1]
```diff
@@ -112,8 +115,8 @@ void mbedtls_aes_free( mbedtls_aes_context *ctx );
  *                 <li>192 bits</li>
  *                 <li>256 bits</li></ul>
  *
- * \return         \c 0 on success or #MBEDTLS_ERR_AES_INVALID_KEY_LENGTH
- *                 on failure.
+ * \return         \c 0 on success.
+ *                 #MBEDTLS_ERR_AES_INVALID_KEY_LENGTH on failure.
```
Comment (hanno-arm) [inline include/mbedtls/aes.h]: Other files use multiple `\return` statements.

Comment (AndrzejKurek) [reply include/mbedtls/aes.h]: This file too.

Comment (RosZad) [reply include/mbedtls/aes.h]: fixed

**Comment by sbutcher-arm** [comment]
@AndrzejKurek - Can you please re-review and approve?

**Pull request merged**